tests/scripts/thread-cert/node.py

```diff
@@ -544,7 +544,13 @@ def remove_domain_prefix(self, prefix):
         self.remove_prefix(prefix)
         self.register_netdata()
 
-    def set_next_dua_response(self, status, iid=None):
+    def set_next_dua_response(self, status: Union[str, int], iid=None):
+        # Convert 5.00 to COAP CODE 160
+        if isinstance(status, str):
+            assert '.' in status
+            status = status.split('.')
+            status = (int(status[0]) << 5) + int(status[1])
```

Comment:
Without much background about coap code, does this formula only works for `5.00 -> 160`? If `status` is other value, is it also correct? If this only works for `5.00`, I think we can just compare the value and set it to `160`.

Reply:
Actually it works for all COAP code, but only 5.00 is useful. I tend to make the code general rather than only handling 5.00 because it's just simple to implement in Python.